internal/ceb/ceb.go

```diff
@@ -61,7 +72,8 @@ type CEB struct {
 	// FIRST child command is ready to be started. This can be closed before
 	// any command is sent to childCmdCh. It indicates that the child process
 	// watcher can begin executing.
-	childReadyCh chan struct{}
+	childReadySent uint32
```

Comment:
I always worry about having a non-pointer sized value mid struct and taking it's address for `atomic` ops. At one point, this could cause issue due to pointer alignment but I'm honestly not sure anymore. I opt usually to just use, in this case, a `*uint32` directly to avoid any potential alignment issues.

Reply:
Yeah I could do that to be safe.

Reply:
We use this pattern a few other places (non-pointer sized value that is passed into `atomic.CompareAndSwap` via `&`). For consistency can we update all of them to match `*type` (ceb, runner, logbuffer, and service_entrypiont)?

Reply:
Done! @krantzinator probably a good idea as we find em.